dev/integration_tests/ios_add2app/README.md

```diff
@@ -0,0 +1,21 @@
+# iOS Add2App Test
+
+This application demonstrates some basic functionality for Add2App,
+along with a native iOS ViewController as a baseline and to demonstrate
+interaction.
+
+The following functionality is currently implemented:
+
+1. A regular iOS view controller, similar to the default `flutter create` template.
```

Comment:
A regular fullscreen instance of a FlutterViewController? Regular iOS view controller sounds non-flutter related.

Reply:
It is.  That one is just a regular iOS implementation for reference.  It's also used in one of the tests for platform interaction.